rootfs/rootfs/etc/rc.d/automount

```diff
@@ -49,7 +49,7 @@ if [ ! -n "$BOOT2DOCKER_DATA" ]; then
                 sleep 0.5
             done
             BOOT2DOCKER_DATA=`echo "${UNPARTITIONED_HD}1"`
-            mkfs.ext4 -L $LABEL $BOOT2DOCKER_DATA
+            mkfs.ext4 -T news -L $LABEL $BOOT2DOCKER_DATA
```

Comment:
Yup, I think 4096 or 8192 would be sensible. I'll update the PR.


On Saturday, February 20, 2016, Tianon Gravi <notifications@github.com>
wrote:

> In rootfs/rootfs/etc/rc.d/automount
> <https://github.com/boot2docker/boot2docker/pull/1135#discussion_r53547073>
> :
>
> > @@ -49,7 +49,7 @@ if [ ! -n "$BOOT2DOCKER_DATA" ]; then
> >                  sleep 0.5
> >              done
> >              BOOT2DOCKER_DATA=`echo "${UNPARTITIONED_HD}1"`
> > -            mkfs.ext4 -L $LABEL $BOOT2DOCKER_DATA
> > +            mkfs.ext4 -T news -L $LABEL $BOOT2DOCKER_DATA
>
> Does our mkfs.ext4 implementation include the -T flag? Does it include
> /etc/mke2fs.conf to define what news means? Would it be safer to instead
> just use -i directly and figure out an OK value? I've had good luck with -i
> 8096 myself generally, but did recently have to reformat my home machine
> to -i 6072 (but granted that I do a lot of work on the Docker Official
> Images, so I have thousands++ of images/layers on a constant, cycling basis
> so I think I'm a bit of an edge case here I think).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/boot2docker/boot2docker/pull/1135/files#r53547073>.
>